build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
 		jcenter()
 	}
 	dependencies {
-		classpath 'com.android.tools.build:gradle:1.5.0'
+		classpath 'com.android.tools.build:gradle:2.0.0'
```

Comment:
@rivaldi8  Thanks for the pr. 
This version upgrade breaks the build, because it depends on a higher version of gradle. 
We will upgrade for the 2.1.x series. But 2.0.x should stay with build tools 1.5.0